packages/block-library/src/navigation/edit/use-imported-classic-menus.js

```diff
@@ -0,0 +1,23 @@
+function useImportedClassicMenus() {
+	if ( ! window.localStorage.getItem( 'classic_menus_imported' ) ) {
+		window.localStorage.setItem(
+			'classic_menus_imported',
+			JSON.stringify( [ ...new Map() ] )
+		);
+	}
+	const importedClassicMenus = new Map(
+		JSON.parse( window.localStorage.getItem( 'classic_menus_imported' ) )
+	);
+
+	function setImportedClassicMenu( navMenuId, classicMenuId ) {
+		importedClassicMenus.set( navMenuId, classicMenuId );
+		window.localStorage.setItem(
+			'classic_menus_imported',
+			JSON.stringify( Array.from( importedClassicMenus.entries() ) )
+		);
+	}
+
+	return { importedClassicMenus, setImportedClassicMenu };
+}
+
+export default useImportedClassicMenus;
```

Comment:
As I'm not 100% sure of using localStorage as a solution, I will remove this and wait for having slug as references to try it.